src/screen/CategoryPage.tsx

```diff
@@ -60,58 +86,91 @@ export function CategoryPage(props: CategoryPageProps) {
       <input
         type="button"
         value={inputType === "input" ? "Add" : "Update"}
+        disabled={createCategory.isLoading ? true : false}
```

Comment:
yang ini bisa disingkat gini

```suggestion
        disabled={createCategory.isLoading}
```

Reply:
updated